basewidget.go

```diff
@@ -0,0 +1,88 @@
+package desktop
+
+import (
```

Comment:
As this is a copy of the baseWidget in fyne can you write that in a comment along with an indicator that we should remove it once the new, better, BaseWidget is complete? linking to https://github.com/fyne-io/fyne/issues/324